Sources/SwiftFusion/Inference/PriorFactor.swift

```diff
@@ -15,9 +15,7 @@
 import PenguinStructures
 
 /// A factor that specifies a prior on a pose.
-public struct PriorFactor<Pose: LieGroup>: LinearizableFactor {
-  public typealias Variables = Tuple1<Pose>
-
+public struct PriorFactor<Pose: LieGroup>: LinearizableFactor1 {
```

Comment:
Should this just be T? And should @ProfFan use this in Chordal rather than make a bespoke prior factor?

Reply:
I believe that we can use this if we see `Matrix3` as GL(3, R) (I considered this approach in #108  but did not pursue it)

Reply:
Yeah we could name it something more generic than `Pose`. I think a full word like `Variable` would be more Swift-style than `T`. I'll leave it as-is in this PR because changing `Pose` to something else is orthogonal to the purpose of this PR.